### PR TITLE
MAINTAINERS: various updates and improvements

### PR DESCRIPTION
**Pull request by nashif**
Various updates to areas and new areas to cover some orphaned files.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Review by fabiobaltieri — DISMISSED** [review]

File: MAINTAINERS.yml
Comment (fabiobaltieri) [inline MAINTAINERS.yml:3203]: is this right?

Comment (nashif) [reply MAINTAINERS.yml:3203]: suggest other labels if  have something in mind, I could not find anything else

Comment (fabiobaltieri) [reply MAINTAINERS.yml:3203]: oh sorry, I thought it was a platform, nah sounds about right

**Review by aescolar — COMMENTED** [review]
No issue from me,
but I guess this requires a TSC vote and approvals from each affected maintainer, or?

**Review by najumon1980 — DISMISSED** [review]

File: MAINTAINERS.yml
Comment (jhedberg) [inline MAINTAINERS.yml:117]: There's a separate section for the acipca west module too, for which I'm a de-facto maintainer (I think I'm also the only one with push access) - could you add me there as well? Thanks.

Comment (nashif) [reply MAINTAINERS.yml:117]: this is different from the west project.

Comment (jhedberg) [reply MAINTAINERS.yml:117]: > this is different from the west project.

I know. I was asking if you could update the section for the west project in the same go, since right now I'm not listed there.

**Review by bjarki-andreasen — DISMISSED** [review]

**Review by peter-mitsis — DISMISSED** [review]

File: MAINTAINERS.yml
Comment (jhedberg) [reply MAINTAINERS.yml:117]: Anyway, not absolutely mandatory to be done in this PR, so I'll approve and let you decide.

**Review by jhedberg — DISMISSED** [review]

File: MAINTAINERS.yml
Comment (dcpleung) [inline MAINTAINERS.yml:3732]: duplicate?

**Review by dcpleung — APPROVED** [review]

**Review by bjarki-andreasen — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

File: MAINTAINERS.yml
Comment (fabiobaltieri) [inline MAINTAINERS.yml:551]: double blank line

Comment (nashif) [reply MAINTAINERS.yml:551]: blank line police

Comment (fabiobaltieri) [reply MAINTAINERS.yml:551]: on nitpicking duties 🚨

**Review by kartben — APPROVED** [review]

File: MAINTAINERS.yml
Comment (jhedberg) [inline MAINTAINERS.yml:120]: Would be good to have @tbursztyka here as well

**Review by stephanosio — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by keith-zephyr — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**
